tortoise.py

```diff
@@ -4,7 +4,10 @@ def main():
     FILENAME = 'tiny.tortoise'
     with open(FILENAME, 'r') as tiny_tortoise:
         program = tiny_tortoise.readlines()
-    program = [i.strip() for i in program]
+    program = [list(i.strip()) for i in program]
```

Comment:
We should have a comment here to explain why we're going this. The next person to read this will appreciate the context.